### DISCUSSION TITLE
how to calculate D-dimensional integral in feyncalc

### DISCUSSION THREAD

**Discussion by iisiaccNewton** [FeynCalc/feyncalc#256] [Q&A]
Can we calculate the following integral in  feyncalc ?
![屏幕截图 2024-03-29 193143](https://github.com/FeynCalc/feyncalc/assets/112673778/7a8512ee-cdb7-4762-b77d-6b88bea54c3e)

**Comment by vsht** [comment]
See

https://feyncalc.github.io/FeynCalcBookDev/FCFeynmanParametrize.html

Am 29.03.24 um 12:33 schrieb asff:

**Reply by iisiaccNewton** [reply]
Thank you for your help. I spent a lot of time looking up the manual but couldn't find it. Now I understand
